src/app/users/users-table.component.html

```diff
@@ -130,6 +130,9 @@
     <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
     <mat-row [ngClass]="{'cursor-pointer':!isDialog}" *matRowDef= "let row; columns: displayedColumns;" (click)="gotoProfileView(row.doc.name)" [ngClass]="{highlight:isSelected(row.doc)}"></mat-row>
   </mat-table>
+  <ng-template #noUserFoundMessage>
```

Comment:
Please move this below the `mat-paginator` element.  That is related to the table and it's confusing seeing this in the middle.

Reply:
Updated both requested changes